besu/src/main/java/org/hyperledger/besu/cli/util/ConfigOptionSearchAndRunHandler.java

```diff
@@ -60,13 +57,23 @@ public ConfigOptionSearchAndRunHandler(
 
   private Optional<File> findConfigFile(
       final ParseResult parseResult, final CommandLine commandLine) {
-    if (parseResult.hasMatchedOption(configFileOptionName)) {
-      final OptionSpec configFileOption = parseResult.matchedOption(configFileOptionName);
+    if (parseResult.hasMatchedOption("--config-file")) {
+      final OptionSpec configFileOption = parseResult.matchedOption("--config-file");
       try {
         return Optional.of(configFileOption.getter().get());
       } catch (final Exception e) {
         throw new ExecutionException(commandLine, e.getMessage(), e);
       }
+    } else if (environment.containsKey("BESU_CONFIG_FILE")) {
```

Comment:
so if I specify --config-file AND the environment variable, the environment variable is ignored. Is this what we want? And if so, do we at least want a warning message for the user?

Reply:
Good spot! Throwing a `ParameterException` instead.